src/lib/navbar/NavItem.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+  export let selected: boolean;
+</script>
+
+<!-- svelte-ignore a11y-clickw-events-have-key-events -->
+<li class="grid grid-cols-[2ch_1fr_3ch] pl-4 cursor-pointer">
```

Comment:
Why the warning ignore? Also, can you explain `[2ch_1fr_3ch]`? I'm not familiar

Reply:
I am not sure about the warning ignore, however  `[2ch_1fr_3ch]` is a tailwind [arbitrary value](https://tailwindcss.com/docs/grid-template-columns#arbitrary-values). In this particular case it is creating 3 columns with sizes 2ch, 1fr and 3ch respectively. This can be moved to a separate CSS file or we can [extend](https://tailwindcss.com/docs/configuration) the `grid-cols-` statement if this particular layout appears more frequently/is reused anywhere else.

Reply:
The warning ignore was used for debug purposes, I'll remove it now since it is not necessary anymore.